lib/UserAgentApplication.ts

```diff
@@ -330,9 +350,8 @@ namespace Msal {
                     "Popup Window is null. This can happen if you are using IE");
                 if (reject) {
                     reject("Popup Window is null. This can happen if you are using IE");
```

Comment:
update the message to mention pop up blockers because it can happen in chrome as well.